src/svm/cctpHelpers.ts

```diff
@@ -118,5 +169,13 @@ export const getMessages = async (txHash: string, srcDomain: number, irisApiUrl:
     }
   }
 
+  // Validate the response structure
+  try {
+    AttestationResponseStruct.assert(attestationResponse);
```

Comment:
Adding runtime type validation with superstruct to ensure the cctp matches the expected schema.